.github/workflows/publish-ext-python-apidocs.yml

```diff
@@ -0,0 +1,61 @@
+name: Publish Python API Docs
+
+# Run when the Python API changes or every month so that the artifact does not expire
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - docs/extensions/**
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+concurrency:
+  group: "apidocs-python"
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    name: Generate Python API docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install tools
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3-pip
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install -r requirements-dev.txt
+          python3 -m pip install onnxruntime>=1.12.0
```

Comment:
Why >=1.12.0?

The version you download doesn't match the version in this repo. 
And the version you download could be arbitrary. It might be just a test build from someone's private branch.  We official docs cannot be based on that.

Reply:
to align with: https://github.com/microsoft/onnxruntime-extensions/blob/main/requirements-dev.txt#L6

i can try to remove the version requirement for now i believe the modules we are creating api-docs for should be supported now regardless of changes in the different latest versions.